utils/servers/grpc.go

```diff
@@ -145,3 +158,35 @@ func (s *grpcServer) Run(ctx context.Context) {
 
 	<-stopped
 }
+
+// injectK8sClient tries to store k8sclient into context. It inspects request's
+// field KubeAuth. If it contains filed Kubeconfig, k8sclient is injected.
+func injectK8sClient(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
```

Comment:
looks like complication of simple code, isn't it?

Reply:
What would you recommend? I can remove this and create the client the same way we do it in other places.
I thought we could use this for every call in the future. So we don't have to call `defer client.Cleanup()` in every API call.

Reply:
Done